tests/integration/test_keeper_s3_snapshot/test.py

```diff
@@ -92,7 +93,7 @@ def get_saved_snapshots():
     # Keeper sends snapshots asynchornously, hence we need to retry.
     @retry(AssertionError, tries=10, delay=2)
     def _check_snapshots():
-        assert set(get_saved_snapshots()) == set(
+        assert set(get_saved_snapshots()) >= set(
```

Comment:
the problem is the extra requests, not the extra nodes
those requests are probably made by server because it automatically uses the embedded Keeper cluster
you can avoid that by setting `use_cluster` to `false` in config
something like this
https://github.com/ClickHouse/ClickHouse/blob/master/tests/integration/test_keeper_broken_logs/configs/enable_keeper2.xml#L3

Let's add it to all 3 Keeper configs for this test